.github/workflows/build_release_assets.yml

```diff
@@ -141,14 +141,8 @@ jobs:
         shell: bash
         run: |
           $PYTHON_CMD -m pip install --upgrade pip
-          $PYTHON_CMD -m pip install --upgrade \
-            pipenv==2023.12.1
-
-          pipenv install --dev
-          pipenv run pip install pyinstaller==6.7.0
-        env:
-          # Disable lock otherwise Windows-only dependencies like colorama are not installed
-          PIPENV_SKIP_LOCK: 1
+          $PYTHON_CMD -m pip install --upgrade pdm
```

Comment:
same, I would say [setup PDM](https://github.com/pdm-project/setup-pdm) + pinned version. In 2 different steps, as PDM is not a dependancy.